tox.ini

```diff
@@ -8,10 +8,21 @@ envlist = py27,py37
 skipsdist = True
 
 [testenv]
+passenv =
+    LD_PRELOAD
+    SSLKEYLOGFILE
```

Comment:
I'm completely sure there's a good reason for these two to be passed down, but it's not clear to me what that reason is. Would you mind commenting on that?

Reply:
These were added so I could debug some really weird TLS errors (which ended up being caused by sendfile). LD_PRELOAD is a standard way of loading an library before the process starts, and SSLKEYLOGFILE is a unofficial standard to write out the TLS config to a file (see https://redflagsecurity.net/2019/03/10/decrypting-tls-wireshark/). I can add comments to the file to that effect if it useful?

Reply:
And the library you were pre-loading was also part of that debugging effort I guess? Yes, if you could just add a small comment above those lines to clarify future readers would be great, thanks!